src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -293,6 +303,24 @@ def upgrade():
         existing_type=postgresql.ARRAY(sa.VARCHAR()),
         nullable=True,
     )
+    op.alter_column(
+        "privacynoticehistory",
+        "displayed_in_privacy_center",
+        existing_type=sa.BOOLEAN(),
+        nullable=True,
+    )
+    op.alter_column(
+        "privacynoticehistory",
+        "displayed_in_overlay",
+        existing_type=sa.BOOLEAN(),
+        nullable=True,
+    )
+    op.alter_column(
+        "privacynoticehistory",
+        "displayed_in_api",
+        existing_type=sa.BOOLEAN(),
+        nullable=True,
+    )
```

Comment:
Making these historical displayed_in* fields nullable - we won't delete them, but leave these values for auditing on early records